src/Tizen.NUI/src/internal/WebView/WebBackForwardList.cs

```diff
@@ -65,6 +65,56 @@ public Uri OriginalUrl
                 return new Uri(Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr));
             }
         }
+
+        /// <summary>
+        /// Delete item.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void DeleteItem()
+        {
+            Interop.WebBackForwardListItem.DeleteItem(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+    }
+
+    /// <summary>
+    /// WebBackForwardCopiedListItems is a class for back-forward copied list item of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardCopiedListItems : Disposable
+    {
+        internal WebBackForwardCopiedListItems(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// <summary>
+        /// Delete copied items.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void DeleteCopiedItems()
+        {
+            Interop.WebBackForwardCopiedListItems.DeleteCopiedItems(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Get the count of copied items.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint GetCopiedItemsCount()
+        {
+            return Interop.WebBackForwardCopiedListItems.GetCopiedItemsCount(SwigCPtr);
+        }
+
+        /// <summary>
+        /// Get the item according to index
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebBackForwardListItem GetItemAtIndexFromCopiedItems(uint index)
+        {
+            System.IntPtr itemPtr = Interop.WebBackForwardCopiedListItems.GetItemAtIndexFromCopiedItems(SwigCPtr, index);
+            return new WebBackForwardListItem(itemPtr, false);
```

Comment:
false --> true.
And DeleteItem need be called when WebBackForwardListItem is disposed.

Reply:
Here ignore the comments above.
but other functions need be changed.

Reply:
done